src/mon/Monitor.cc

```diff
@@ -375,28 +375,45 @@ int Monitor::do_admin_command(
     start_election();
     elector.stop_participating();
     out << "stopped responding to quorum, initiated new election";
-  } else if (command == "ops") {
```

Comment:
~~Oops, I would prefer to keep this option due to compatibility reasons.
Perhaps we can warn / deprecate.~~

UPDATE: re-added later.